src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -179,10 +179,12 @@ function PolicyDistanceRatesPage({policy, route}: PolicyDistanceRatesPageProps)
     };
 
     const toggleAllRates = () => {
-        if (selectedDistanceRates.length === Object.values(customUnitRates).length) {
+        const allSelectableRates = Object.values(customUnitRates).filter((rate) => rate.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
```

Comment:
minor fix to toggleAllRates functionality. It wasn't working properly in offline mode